monitor/log_sample_spike.txt

```diff
@@ -0,0 +1,29 @@
+date=2019-04-18 total-logs=15097
```

Comment:
This looks like test data - put it in the `tests` directory under a subdirectory of `data` or something.

Reply:
Done